appveyor.yml

```diff
@@ -0,0 +1,14 @@
+init:
```

Comment:
Yes please. I think it does help speeding up our builds. Plus the expanding % thing at the top of our build logs looks kinda ridiculous